src/components/createCategory.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import RaisedButton from 'material-ui/RaisedButton';
+import TextField from 'material-ui/TextField';
+
+
+import axios from 'axios';
+import './styling.css';
+
+import style5 from './styling';
+import style4 from './styling';
+import style from './styling';
+
+
+
+
+class CreateCategory extends Component {
+    constructor(props){
+        super(props);
+        this.state={
+          open: false,
+          category_title:'',
+          category_description:''
+        };
+      } 
+
+      handleClick = (e) =>{
+        e.preventDefault();
+        const CreateCategoryUrl='http://127.0.0.1:5000/create_category';
```

Comment:
Use "http://127.0.0.1:5000" as base URL ten append other parameters for the different components